astropy/wcs/tests/test_wcs.py

```diff
@@ -1541,3 +1541,17 @@ def test_time_axis_selection(tab_wcs_2di_f):
     assert list(w.sub([wcs.WCSSUB_TIME]).wcs.ctype) == ['TIME']
     assert (w.wcs_pix2world([[1, 2, 3]], 0)[0, 2] ==
             w.sub([wcs.WCSSUB_TIME]).wcs_pix2world([[3]], 0)[0, 0])
+
+
+@pytest.mark.skipif(
+    _WCSLIB_VER < Version('7.8'),
+    reason="TIME axis extraction only works with wcslib 7.8 or later"
+)
+def test_temporal(tab_wcs_2di_f):
```

Comment:
Did you intend to use tab_wcw_2di_f ?

Reply:
And in the above test as well, I missed it before.

Reply:
A mistake